src/micro_http/src/server.rs

```diff
@@ -463,7 +463,8 @@ impl HttpServer {
     /// Accepts a new incoming connection and adds it to the `epoll` notification structure.
     ///
     /// # Errors
-    /// `IOError` is returned when an `epoll::ctl` operation fails.
+    /// `IOError` is returned when an `epoll::ctl`, socket accept, set stream non-blocking operations fails.
```

Comment:
I d change this comment to something more generic:
`IOError is returned when socket or epoll operations fail.`